packages/core/src/indexing/postgres/__tests__/postgres-index-api.test.ts

```diff
@@ -147,17 +161,52 @@ describe('indexStream', () => {
   test('new stream', async () => {
     const now = new Date()
     await indexApi.indexStream(STREAM_CONTENT)
-    const result: Array<any> = await dbConnection.from(`${MODELS_TO_INDEX[0]}`).select('*')
+    let result: Array<any> = await dbConnection
+      .from(`${MODELS_TO_INDEX[0]}`)
+      .select(
+        dbConnection.raw(
+          `*, stream_content->'settings'->'dark_mode' AS dark_mode, stream_content->>'id' AS id`
+        )
+      )
     expect(result.length).toEqual(1)
     const raw = result[0]
     expect(raw.stream_id).toEqual(STREAM_ID_B)
     expect(raw.controller_did).toEqual(CONTROLLER)
+    expect(raw.stream_content).toEqual(STREAM_TEST_DATA_PROFILE)
+    expect(raw.tip).toEqual(FAKE_CID.toString())
     expect(raw.last_anchored_at).toBeNull()
     expect(raw.first_anchored_at).toBeNull()
     const createdAt = new Date(raw.created_at)
     const updatedAt = new Date(raw.updated_at)
     expect(closeDates(createdAt, now)).toBeTruthy()
     expect(closeDates(updatedAt, now)).toBeTruthy()
+    expect(raw.dark_mode).toEqual(STREAM_TEST_DATA_PROFILE.settings.dark_mode)
+    expect(raw.id).toEqual(STREAM_TEST_DATA_PROFILE.id)
+
+    // create index on jsonb content
+    result = await dbConnection.raw(
```

Comment:
seems like creating the index should happen in the test fixture setup

Reply:
I can push it up/split - I mainly kept it together here since we already are populating and confirming the data but yeah I can push that up into the fixture.

Reply:
I looked into the fixture creation but given how it is structured right now with tables being dropped before each run I made it a separate test instead. 

As I wrote in the other comment it's mainly a confirmation that the index is indeed invoked when queried BUT I'm honestly not sure if this test overall adds any value given that it only tests a base db function and not really anything indexing feature related.